README.md

```diff
@@ -96,6 +96,32 @@ drwxr-xr-x 3 root root     4096 Apr  4 10:15 ..
 
 WARNING: If uefi entry was already created by uefi-mkconfig for this kernel before `.ignore` file creation. **Its uefi entry will be deleted!**
 
+## Backup UEFI entry creation
+
+uefi-mkconfig can automatically create backup uefi entry at position 0200.
+This entry **will not** be automatically deleted and **will not** be added to the bootorder.
+Besides these two special rules, the entry creation itself is identical to the normal processs.
+
+**Only one kernel image** can be designated as backup. If multiple one are marked as backup, kernel of the most recent version is chosen.
+
+To designate kernel image as backup, create an empty file in the same directory as the kernel image itsel, name it the same way but add suffix ".uefibackup"
```

Comment:
Great! I like this idea

Reply:
Maybe we shouldn't use 0200 though, that makes it impractical if we in the future want to expand the managed range. If the backup entry is the first (0100) instead, we avoid that problem.

Reply:
The problem with this approach is migrating users to it. I would need to add some migration logic to check if entry on 0100 is actually the new backup entry or just ordinary entry from older version.

If we need to expand the managed range for example from 0100 to 0300 we can move the backup entry to 0300 but still just leave 0200 whitelisted for backward compatibility.

Reply:
Don't we need logic to double check the back-up entry location anyway? There might also be some pre-exisiting non-uefi-mkconfig entry there.

Reply:
The easiest way might be to simply check if the efi file belonging to our back-up entry contains ".uefibackup" or not. If not then it is either made by older uefi-mkconfig or by something else, and we can remove it.

Reply:
Indeed, you are right. 

In this case, I'll also create checking logic for wiping that range. I don't have a problem with nuking anything that uefi-mkconfig has created, since it will be regenerated anyway.

But if something has accidentally created entry in that range we will nuke it as well.

First thing that comes to my mind is some kind of "stamp" in the entry label that would mark it as ours.
Something like:

```
UMC 6.9.6-gentoo-dist-old on EFI System Partition
```

Nuking that entire range indiscriminately is kinda irresponsible.

Reply:
> The easiest way might be to simply check if the efi file belonging to an existing back-up entry contains ".uefibackup" or not. If not then it is either made by older uefi-mkconfig or by something else, and we can remove it.

Good idea!

Reply:
However If we would go the "stamp" route i would just do something like this for every backup entry:

```
UMC BCK 6.9.6-gentoo-dist-old on EFI System Partition
```

Reply:
> But if something has accidentally created entry in that range we will nuke it as well.

Indeed, this is likely to happen at some point because I think `efibootmgr --create` uses by default the highest occupied number +1. So if uefi-mkconfig made entries in e.g. 100 to 110, then e.g. `bootctl --install` would make an entry for systemd-boot at 111, which is in our range. 

> First thing that comes to my mind is some kind of "stamp" in the entry label that would mark it as ours.
Something like:

Yes that seems like a good idea to me!

Reply:
Now that I'm thinking about it. Existence of UMCB stamp technically makes setting precise ID 0100 for backup entry useless.

We dont need it because we can identify backup entry by something else